frontend/src/modules/Tables/views/TableView.js

```diff
@@ -27,12 +27,15 @@ import {
   useSettings
 } from 'design';
 import { SET_ERROR, useDispatch } from 'globalErrors';
-import { deleteDatasetTable, getDatasetTable, useClient } from 'services';
+import { deleteDatasetTable, useClient } from 'services';
```

Comment:
Why is the `deleteDatasetTable` part of the global services? Shouldn't it be part of the table/services